internal/lintcontext/context.go

```diff
@@ -23,22 +23,37 @@ type InvalidObject struct {
 }
 
 // A LintContext represents the context for a lint run.
-type LintContext struct {
+type LintContext interface {
+	GetObjects() []Object
+	GetInvalidObjects() []InvalidObject
```

Comment:
We should keep the name `Objects()` and `InvalidObjects()`, in keeping with Go convention (https://golang.org/doc/effective_go.html#Getters)